scripts/add_electricity.py

```diff
@@ -0,0 +1,404 @@
+import os
+from _helpers import configure_logging, sets_path_to_root
+
+import pypsa
+import pandas as pd
+import xarray as xr
+import matplotlib.pyplot as plt
+import numpy as np
+import powerplantmatching as pm
+
+
+"""
+Adds electrical generators, load and storage units to a base network.
```

Comment:
Similarly to the other functions, this block """...""" shall be moved even before the imports

This applies to all function, I won't repeat

Reply:
I made the correction in add_electricity, create_network and solve_network.